<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создать/Изменить</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string> Создать:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTableWidget" name="tableWidgetCreate"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="btn_create">
      <property name="text">
       <string>Создать</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Изменить:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QComboBox" name="comboBox">
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btn_load">
      <property name="text">
       <string>Загрузить элемент</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QTableWidget" name="tableWidgetEdit"/>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="btn_edit">
      <property name="text">
       <string>Изменить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
